vector/src/test/java/im/vector/app/features/location/LocationDataTest.kt

```diff
@@ -57,4 +60,16 @@ class LocationDataTest {
         parseGeo("ge o:12.34,56.78;13.56").shouldBeNull()
         parseGeo("geo :12.34,56.78;13.56").shouldBeNull()
     }
+
+    @Test
+    fun genericLocationTest() {
+        val contentWithNullAsset = MessageLocationContent(body = "", geoUri = "", locationAsset = null)
+        contentWithNullAsset.isGenericLocation() shouldBeEqualTo(false)
```

Comment:
Nit: better to use `.shouldBeFalse()` and `.shouldBeTrue()`

Reply:
Done. I didn't know these exist :O